src/lib/index/wb_cache.hpp

```diff
@@ -59,19 +59,18 @@ class IndexWBCache : public IndexWBCacheBase {
     //////////////////// CP Related API section /////////////////////////////////
```

Comment:
A question for return type of create_chain, shoudn't it be std::pair since there is only two elements in there?

Reply:
Ack